pom.xml

```diff
@@ -103,6 +104,18 @@
                 <artifactId>jnosql-mapping-key-value</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
I've updated the pom parent; we don't need this anymore.
Could you check it, please?